src/extensions/curation/curation.tsx

```diff
@@ -13,7 +13,7 @@ export function activate(context: ExtensionContext) {
         priority: 1000,
         component: CurationSortingView,
         props: {
-            height: 500
+            height: 1000
```

Comment:
increasing the hard-coded height. Will need to remedy. But curation.tsx will be probably be replaced by mountainview anyway.